src/frontends/paddle/tests/test_models/gen_scripts/generate_unique.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+#
+# unique paddle model generator
+#
+import numpy as np
+from save_model import saveModel
+import paddle
+import sys
+
+data_type = "float32"
+
+
+def unique(
+    name: str,
+    x,
+    reture_index=False,
+    reture_inverse=False,
+    reture_counts=False,
+    dtype="int64",
+    axes=None,
```

Comment:
Can we leave these parameter's name as origin one, which can be loaded to ```paddle.unique``` directly

Reply:
Already fixed.